ui/src/tempVars/components/TagValuesTemplateBuilder.tsx

```diff
@@ -0,0 +1,308 @@
+import React, {PureComponent} from 'react'
+import _ from 'lodash'
+
+import {
```

Comment:
to give some color to why i've made comments about ordering of imports.. the general convention i've followed and find useful is, by line-separated section:
1. packages/libraries (react, redux, lodash, etc.)
1. containers & components (including HOCs)
1. actions/apis
1. data transformation (parsers, etc.)
1. error stuff
1. types